### PR TITLE
Add basic grid based movement

### PR DESCRIPTION
**Pull request by Dennisbonke**
This PR adds a basic grid (square, 6x3), with a nice white tictac capsule that can move to any adjacent tile (including diagonals). Checking is applied to make sure that the requested move is valid. Move the tictac by left clicking on adjacent cells.

### CODE REVIEW

File: Assets/Scripts/Grid.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	public void GetXZ(Vector3 worldPosition, out int x, out int z) {
+		if(DebugGrid) {
+			Debug.Log("GetXZ got a vector 3 with x: " + worldPosition.x + ", y: " + worldPosition.y + " and z: " + worldPosition.z + "!");
+			Debug.Log("GetXZ got a vector 3 (including origin) with x: " + (worldPosition - originPosition).x + ", y: " + (worldPosition - originPosition).y + " and z: " + (worldPosition - originPosition).z + "!");
+		}
+		x = Mathf.FloorToInt((worldPosition - originPosition).x / cellSize);
+		z = Mathf.FloorToInt((worldPosition - originPosition).z / cellSize);
+	}
+
+	// Graciously taken from the interwebs, draws a line like Debug.DrawLine does
```
Comment (Westlanderz) [inline Assets/Scripts/Grid.cs:52]: End with a sentence with a `.`

**Review by Westlanderz — CHANGES_REQUESTED** [review]
Except for that one sentence that does not end with the `.` the PR looks good and ready to merge.

File: Assets/Scripts/Grid.cs (hunk 1 above)
Comment (Dennisbonke) [reply Assets/Scripts/Grid.cs]: Solved.

**Review by Westlanderz — CHANGES_REQUESTED** [review]
Still missing some `.`, we missed them last review.

**Review by Westlanderz — APPROVED** [review]
LGTM

**Pull request merged**